frontend/packages/ux-editor/src/components/config/componentSpecificContent/Summary2/Override/Summary2OverrideEntry.tsx

```diff
@@ -8,6 +8,9 @@ import { useAppContext, useComponentTypeName } from '../../../../../hooks';
 import { useFormLayoutsQuery } from '../../../../../hooks/queries/useFormLayoutsQuery';
 import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
 import { Summmary2ComponentReferenceSelector } from '../Summary2ComponentReferenceSelector';
+import { ComponentType } from 'app-shared/types/ComponentType';
+import { BASE_CONTAINER_ID } from 'app-shared/constants';
+import { Summary2OverrideDisplaytype } from './Summary2OverrideDisplaytype';
```

Comment:
Should we have uppercase `T` in `type` here? 🤔

Reply:
fixed